pkg/storage/snapshot/restore.go

```diff
@@ -765,22 +770,23 @@ func (t *vmRestoreTarget) createDataVolume(dvt kubevirtv1.DataVolumeTemplateSpec
 	return true, nil
 }
 
-func (t *vmRestoreTarget) Own(obj metav1.Object) {
+func (t *vmRestoreTarget) Own(obj metav1.Object) bool {
 	if !t.doesTargetVMExist() {
-		return
+		return false
 	}
 
-	b := true
 	obj.SetOwnerReferences([]metav1.OwnerReference{
 		{
 			APIVersion:         kubevirtv1.GroupVersion.String(),
 			Kind:               "VirtualMachine",
 			Name:               t.vm.Name,
 			UID:                t.vm.UID,
-			Controller:         &b,
-			BlockOwnerDeletion: &b,
+			Controller:         ptr.To(true),
```

Comment:
You should probably use kubevirt pointer pkg "kubevirt.io/kubevirt/pkg/pointer" and then it will be pointer.P(true)